xml/System.Reflection/MethodSemanticsAttributes.xml

```diff
@@ -71,7 +82,16 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Used to read the value of the property.</summary>
```

Comment:
Is "Used to" necessary (or consistent with related docs)? Seems like "Reads the value..." is a bit cleaner. (also applies to other doc below)

Reply:
Agreed. I'll modify it:

```suggestion
        <summary>Reads the value of the property.</summary>
```